2024-03-22T11:39:40.414434171Z    _____                               
2024-03-22T11:39:40.422669879Z   /  _  \ __________ _________   ____  
2024-03-22T11:39:40.422683279Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T11:39:40.422686879Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T11:39:40.422689779Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T11:39:40.422692979Z         \/      \/                  \/ 
2024-03-22T11:39:40.422695779Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T11:39:40.422698879Z 
2024-03-22T11:39:40.422701579Z Documentation: http://aka.ms/webapp-linux
2024-03-22T11:39:40.422704379Z Python 3.11.7
2024-03-22T11:39:40.422707079Z Note: Any data outside '/home' is not persisted
2024-03-22T11:39:42.505993212Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T11:39:42.635921017Z App Command Line not configured, will attempt auto-detect
2024-03-22T11:39:42.645126137Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T11:39:42.665902710Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T11:39:42.676072043Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T11:39:42.677893067Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T11:39:42.685302364Z Output is compressed. Extracting it...
2024-03-22T11:39:42.692501059Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T11:40:14.285534692Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T11:40:14.306015958Z Detected an app based on Flask
2024-03-22T11:40:14.306046459Z Generating `gunicorn` command for 'app:app'
2024-03-22T11:40:14.316246391Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T11:40:14.427657939Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T11:40:14.427688740Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T11:40:15.570612996Z [2024-03-22 11:40:15 +0000] [73] [INFO] Starting gunicorn 21.2.0
2024-03-22T11:40:15.680888729Z [2024-03-22 11:40:15 +0000] [73] [INFO] Listening at: http://0.0.0.0:8000 (73)
2024-03-22T11:40:15.680936430Z [2024-03-22 11:40:15 +0000] [73] [INFO] Using worker: sync
2024-03-22T11:40:15.702136706Z [2024-03-22 11:40:15 +0000] [76] [INFO] Booting worker with pid: 76
2024-03-22T11:40:26.517377506Z [2024-03-22 11:40:26 +0000] [76] [ERROR] Exception in worker process
2024-03-22T11:40:26.517405107Z Traceback (most recent call last):
2024-03-22T11:40:26.517409707Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T11:40:26.517423407Z     worker.init_process()
2024-03-22T11:40:26.517426707Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T11:40:26.517429907Z     self.load_wsgi()
2024-03-22T11:40:26.517436307Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T11:40:26.517439507Z     self.wsgi = self.app.wsgi()
2024-03-22T11:40:26.517449307Z                 ^^^^^^^^^^^^^^^
2024-03-22T11:40:26.517452207Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T11:40:26.517455507Z     self.callable = self.load()
2024-03-22T11:40:26.517458508Z                     ^^^^^^^^^^^
2024-03-22T11:40:26.517461508Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T11:40:26.517464608Z     return self.load_wsgiapp()
2024-03-22T11:40:26.517467408Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T11:40:26.517470308Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T11:40:26.517473408Z     return util.import_app(self.app_uri)
2024-03-22T11:40:26.517476308Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:40:26.517479508Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T11:40:26.517482708Z     mod = importlib.import_module(module)
2024-03-22T11:40:26.517485708Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:40:26.517488708Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T11:40:26.517491908Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T11:40:26.517494908Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:40:26.517497908Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T11:40:26.517501908Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T11:40:26.517505108Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T11:40:26.517508308Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T11:40:26.517511308Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T11:40:26.517514508Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T11:40:26.517517608Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T11:40:26.517521608Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T11:40:26.517524708Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T11:40:26.527145233Z [2024-03-22 11:40:26 +0000] [76] [INFO] Worker exiting (pid: 76)
2024-03-22T11:40:27.303060508Z [2024-03-22 11:40:27 +0000] [73] [ERROR] Worker (pid:76) exited with code 3
2024-03-22T11:40:27.303944320Z [2024-03-22 11:40:27 +0000] [73] [ERROR] Shutting down: Master
2024-03-22T11:40:27.304286024Z [2024-03-22 11:40:27 +0000] [73] [ERROR] Reason: Worker failed to boot.

2024-03-22T11:47:28.652754351Z    _____                               
2024-03-22T11:47:28.652783951Z   /  _  \ __________ _________   ____  
2024-03-22T11:47:28.652787852Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T11:47:28.652790952Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T11:47:28.652793952Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T11:47:28.652797352Z         \/      \/                  \/ 
2024-03-22T11:47:28.652800152Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T11:47:28.652803052Z 
2024-03-22T11:47:28.652805752Z Documentation: http://aka.ms/webapp-linux
2024-03-22T11:47:28.652808452Z Python 3.11.7
2024-03-22T11:47:28.652811152Z Note: Any data outside '/home' is not persisted
2024-03-22T11:47:31.353012061Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T11:47:31.525749639Z App Command Line not configured, will attempt auto-detect
2024-03-22T11:47:31.525781940Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T11:47:31.707311333Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T11:47:31.718094276Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T11:47:31.786343676Z Output is compressed. Extracting it...
2024-03-22T11:47:31.786364476Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T11:47:31.786432577Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T11:47:53.417547939Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T11:47:53.428449566Z Detected an app based on Flask
2024-03-22T11:47:53.428826670Z Generating `gunicorn` command for 'app:app'
2024-03-22T11:47:53.435093243Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T11:47:53.566351870Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T11:47:53.567359182Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T11:47:54.913599240Z [2024-03-22 11:47:54 +0000] [72] [INFO] Starting gunicorn 21.2.0
2024-03-22T11:47:54.938210526Z [2024-03-22 11:47:54 +0000] [72] [INFO] Listening at: http://0.0.0.0:8000 (72)
2024-03-22T11:47:54.939407040Z [2024-03-22 11:47:54 +0000] [72] [INFO] Using worker: sync
2024-03-22T11:47:54.954686918Z [2024-03-22 11:47:54 +0000] [75] [INFO] Booting worker with pid: 75
2024-03-22T11:48:07.287414454Z [2024-03-22 11:48:07 +0000] [75] [ERROR] Exception in worker process
2024-03-22T11:48:07.287455554Z Traceback (most recent call last):
2024-03-22T11:48:07.287461354Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T11:48:07.287465154Z     worker.init_process()
2024-03-22T11:48:07.287481055Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T11:48:07.287484655Z     self.load_wsgi()
2024-03-22T11:48:07.287487655Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T11:48:07.287491155Z     self.wsgi = self.app.wsgi()
2024-03-22T11:48:07.287494155Z                 ^^^^^^^^^^^^^^^
2024-03-22T11:48:07.287497255Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T11:48:07.287500355Z     self.callable = self.load()
2024-03-22T11:48:07.287503255Z                     ^^^^^^^^^^^
2024-03-22T11:48:07.287506155Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T11:48:07.287509455Z     return self.load_wsgiapp()
2024-03-22T11:48:07.287512455Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:07.287515455Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T11:48:07.287518655Z     return util.import_app(self.app_uri)
2024-03-22T11:48:07.287521555Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:07.287524555Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T11:48:07.287527655Z     mod = importlib.import_module(module)
2024-03-22T11:48:07.287530655Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:07.287533755Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T11:48:07.287536955Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T11:48:07.287540055Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:07.287543055Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T11:48:07.287546756Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T11:48:07.287549856Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T11:48:07.287552956Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T11:48:07.287555956Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T11:48:07.287559256Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T11:48:07.287562556Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T11:48:07.287566956Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T11:48:07.287570056Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T11:48:07.287576456Z [2024-03-22 11:48:07 +0000] [75] [INFO] Worker exiting (pid: 75)
2024-03-22T11:48:07.873378691Z [2024-03-22 11:48:07 +0000] [72] [ERROR] Worker (pid:75) exited with code 3
2024-03-22T11:48:07.881436401Z [2024-03-22 11:48:07 +0000] [72] [ERROR] Shutting down: Master
2024-03-22T11:48:07.881453301Z [2024-03-22 11:48:07 +0000] [72] [ERROR] Reason: Worker failed to boot.

2024-03-22T11:48:18.925730141Z    _____                               
2024-03-22T11:48:18.925754442Z   /  _  \ __________ _________   ____  
2024-03-22T11:48:18.925758742Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T11:48:18.925761842Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T11:48:18.925764742Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T11:48:18.925768042Z         \/      \/                  \/ 
2024-03-22T11:48:18.925770842Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T11:48:18.925773642Z 
2024-03-22T11:48:18.925776242Z Documentation: http://aka.ms/webapp-linux
2024-03-22T11:48:18.925779142Z Python 3.11.7
2024-03-22T11:48:18.925781842Z Note: Any data outside '/home' is not persisted
2024-03-22T11:48:20.281622060Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T11:48:20.492214543Z App Command Line not configured, will attempt auto-detect
2024-03-22T11:48:20.492415946Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T11:48:20.762664194Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T11:48:20.795536460Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T11:48:20.803547073Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T11:48:20.803573774Z Output is compressed. Extracting it...
2024-03-22T11:48:20.804883492Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T11:48:41.756372459Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T11:48:41.768102916Z Detected an app based on Flask
2024-03-22T11:48:41.769235531Z Generating `gunicorn` command for 'app:app'
2024-03-22T11:48:41.777656344Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T11:48:41.916868213Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T11:48:41.916899313Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T11:48:43.095933938Z [2024-03-22 11:48:43 +0000] [70] [INFO] Starting gunicorn 21.2.0
2024-03-22T11:48:43.124632024Z [2024-03-22 11:48:43 +0000] [70] [INFO] Listening at: http://0.0.0.0:8000 (70)
2024-03-22T11:48:43.125885340Z [2024-03-22 11:48:43 +0000] [70] [INFO] Using worker: sync
2024-03-22T11:48:43.136453782Z [2024-03-22 11:48:43 +0000] [73] [INFO] Booting worker with pid: 73
2024-03-22T11:48:52.446793515Z [2024-03-22 11:48:52 +0000] [73] [ERROR] Exception in worker process
2024-03-22T11:48:52.446847916Z Traceback (most recent call last):
2024-03-22T11:48:52.446852916Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T11:48:52.446865616Z     worker.init_process()
2024-03-22T11:48:52.446868916Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T11:48:52.446872116Z     self.load_wsgi()
2024-03-22T11:48:52.446875116Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T11:48:52.446878616Z     self.wsgi = self.app.wsgi()
2024-03-22T11:48:52.446881616Z                 ^^^^^^^^^^^^^^^
2024-03-22T11:48:52.446884516Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T11:48:52.446887616Z     self.callable = self.load()
2024-03-22T11:48:52.446896517Z                     ^^^^^^^^^^^
2024-03-22T11:48:52.446899817Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T11:48:52.446903017Z     return self.load_wsgiapp()
2024-03-22T11:48:52.446906117Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:52.446909017Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T11:48:52.446912117Z     return util.import_app(self.app_uri)
2024-03-22T11:48:52.446915017Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:52.446918217Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T11:48:52.446921517Z     mod = importlib.import_module(module)
2024-03-22T11:48:52.446924517Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:52.446927517Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T11:48:52.446930517Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T11:48:52.446933417Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:48:52.446936317Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T11:48:52.446939717Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T11:48:52.446943017Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T11:48:52.446946217Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T11:48:52.446949417Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T11:48:52.446952517Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T11:48:52.446955617Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T11:48:52.446959717Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T11:48:52.446962717Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T11:48:52.457111746Z [2024-03-22 11:48:52 +0000] [73] [INFO] Worker exiting (pid: 73)
2024-03-22T11:48:53.161291225Z [2024-03-22 11:48:53 +0000] [70] [ERROR] Worker (pid:73) exited with code 3
2024-03-22T11:48:53.170205747Z [2024-03-22 11:48:53 +0000] [70] [ERROR] Shutting down: Master
2024-03-22T11:48:53.170482550Z [2024-03-22 11:48:53 +0000] [70] [ERROR] Reason: Worker failed to boot.

2024-03-22T11:49:09.867559182Z    _____                               
2024-03-22T11:49:09.867585182Z   /  _  \ __________ _________   ____  
2024-03-22T11:49:09.867589182Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T11:49:09.867592182Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T11:49:09.867595082Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T11:49:09.867598182Z         \/      \/                  \/ 
2024-03-22T11:49:09.867601182Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T11:49:09.867603882Z 
2024-03-22T11:49:09.867606482Z Documentation: http://aka.ms/webapp-linux
2024-03-22T11:49:09.867609283Z Python 3.11.7
2024-03-22T11:49:09.867612083Z Note: Any data outside '/home' is not persisted
2024-03-22T11:49:12.270389793Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T11:49:12.471856469Z App Command Line not configured, will attempt auto-detect
2024-03-22T11:49:12.472670280Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T11:49:12.525070875Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T11:49:12.531879866Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T11:49:12.533220084Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T11:49:12.533607589Z Output is compressed. Extracting it...
2024-03-22T11:49:12.543295517Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T11:49:34.521155190Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T11:49:34.530967122Z Detected an app based on Flask
2024-03-22T11:49:34.530978722Z Generating `gunicorn` command for 'app:app'
2024-03-22T11:49:34.525321146Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T11:49:34.634810411Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T11:49:34.634838212Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T11:49:35.790672078Z [2024-03-22 11:49:35 +0000] [72] [INFO] Starting gunicorn 21.2.0
2024-03-22T11:49:35.828502284Z [2024-03-22 11:49:35 +0000] [72] [INFO] Listening at: http://0.0.0.0:8000 (72)
2024-03-22T11:49:35.829177693Z [2024-03-22 11:49:35 +0000] [72] [INFO] Using worker: sync
2024-03-22T11:49:35.840558046Z [2024-03-22 11:49:35 +0000] [75] [INFO] Booting worker with pid: 75
2024-03-22T11:49:45.390926371Z [2024-03-22 11:49:45 +0000] [75] [ERROR] Exception in worker process
2024-03-22T11:49:45.390980971Z Traceback (most recent call last):
2024-03-22T11:49:45.390986071Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T11:49:45.390997672Z     worker.init_process()
2024-03-22T11:49:45.391000972Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T11:49:45.391004072Z     self.load_wsgi()
2024-03-22T11:49:45.391007072Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T11:49:45.391010172Z     self.wsgi = self.app.wsgi()
2024-03-22T11:49:45.391013072Z                 ^^^^^^^^^^^^^^^
2024-03-22T11:49:45.391015972Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T11:49:45.391019072Z     self.callable = self.load()
2024-03-22T11:49:45.391028472Z                     ^^^^^^^^^^^
2024-03-22T11:49:45.391031472Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T11:49:45.391034572Z     return self.load_wsgiapp()
2024-03-22T11:49:45.391037472Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T11:49:45.391040272Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T11:49:45.391043572Z     return util.import_app(self.app_uri)
2024-03-22T11:49:45.391046472Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:49:45.391049672Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T11:49:45.391052872Z     mod = importlib.import_module(module)
2024-03-22T11:49:45.391055972Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:49:45.391058972Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T11:49:45.391062072Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T11:49:45.391064972Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T11:49:45.391067973Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T11:49:45.391071473Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T11:49:45.391074773Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T11:49:45.391077873Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T11:49:45.391080973Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T11:49:45.391084173Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T11:49:45.391087273Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T11:49:45.391091273Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T11:49:45.391094273Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T11:49:45.399470082Z [2024-03-22 11:49:45 +0000] [75] [INFO] Worker exiting (pid: 75)
2024-03-22T11:49:46.113992929Z [2024-03-22 11:49:46 +0000] [72] [ERROR] Worker (pid:75) exited with code 3
2024-03-22T11:49:46.193570370Z [2024-03-22 11:49:46 +0000] [72] [ERROR] Shutting down: Master
2024-03-22T11:49:46.193600270Z [2024-03-22 11:49:46 +0000] [72] [ERROR] Reason: Worker failed to boot.

2024-03-22T12:09:47.538682544Z    _____                               
2024-03-22T12:09:47.538711645Z   /  _  \ __________ _________   ____  
2024-03-22T12:09:47.538715545Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T12:09:47.538718545Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T12:09:47.538721945Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T12:09:47.538725045Z         \/      \/                  \/ 
2024-03-22T12:09:47.538727945Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T12:09:47.538730745Z 
2024-03-22T12:09:47.538733445Z Documentation: http://aka.ms/webapp-linux
2024-03-22T12:09:47.538736145Z Python 3.11.7
2024-03-22T12:09:47.538738845Z Note: Any data outside '/home' is not persisted
2024-03-22T12:09:49.503763731Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T12:09:49.664935646Z App Command Line not configured, will attempt auto-detect
2024-03-22T12:09:49.664961046Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T12:09:49.736616986Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T12:09:49.739644026Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T12:09:49.741389849Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T12:09:49.744120285Z Output is compressed. Extracting it...
2024-03-22T12:09:49.745206699Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T12:10:18.609539640Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T12:10:18.611293365Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T12:10:18.611309865Z Detected an app based on Flask
2024-03-22T12:10:18.611313865Z Generating `gunicorn` command for 'app:app'
2024-03-22T12:10:18.792764394Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T12:10:18.792795394Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T12:10:20.631401716Z [2024-03-22 12:10:20 +0000] [72] [INFO] Starting gunicorn 21.2.0
2024-03-22T12:10:20.669929353Z [2024-03-22 12:10:20 +0000] [72] [INFO] Listening at: http://0.0.0.0:8000 (72)
2024-03-22T12:10:20.671117869Z [2024-03-22 12:10:20 +0000] [72] [INFO] Using worker: sync
2024-03-22T12:10:20.691955060Z [2024-03-22 12:10:20 +0000] [75] [INFO] Booting worker with pid: 75
2024-03-22T12:10:32.789182885Z [2024-03-22 12:10:32 +0000] [75] [ERROR] Exception in worker process
2024-03-22T12:10:32.789219286Z Traceback (most recent call last):
2024-03-22T12:10:32.789223886Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T12:10:32.789293787Z     worker.init_process()
2024-03-22T12:10:32.789298687Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T12:10:32.789301987Z     self.load_wsgi()
2024-03-22T12:10:32.789304987Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T12:10:32.789308087Z     self.wsgi = self.app.wsgi()
2024-03-22T12:10:32.789311187Z                 ^^^^^^^^^^^^^^^
2024-03-22T12:10:32.789314387Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T12:10:32.789317487Z     self.callable = self.load()
2024-03-22T12:10:32.789340288Z                     ^^^^^^^^^^^
2024-03-22T12:10:32.789343688Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T12:10:32.789346688Z     return self.load_wsgiapp()
2024-03-22T12:10:32.789349688Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T12:10:32.789352488Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T12:10:32.789355588Z     return util.import_app(self.app_uri)
2024-03-22T12:10:32.789358688Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:10:32.789361588Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T12:10:32.789364688Z     mod = importlib.import_module(module)
2024-03-22T12:10:32.789367788Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:10:32.789371288Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T12:10:32.789374588Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T12:10:32.789377588Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:10:32.789380488Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T12:10:32.789383988Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T12:10:32.789387088Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T12:10:32.789390188Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T12:10:32.789393388Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T12:10:32.789411988Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T12:10:32.789416389Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T12:10:32.789420789Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T12:10:32.789423789Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T12:10:32.789430989Z [2024-03-22 12:10:32 +0000] [75] [INFO] Worker exiting (pid: 75)
2024-03-22T12:10:34.234114231Z [2024-03-22 12:10:34 +0000] [72] [ERROR] Worker (pid:75) exited with code 3
2024-03-22T12:10:34.235261446Z [2024-03-22 12:10:34 +0000] [72] [ERROR] Shutting down: Master
2024-03-22T12:10:34.235632952Z [2024-03-22 12:10:34 +0000] [72] [ERROR] Reason: Worker failed to boot.

2024-03-22T12:17:59.965695799Z    _____                               
2024-03-22T12:17:59.965721799Z   /  _  \ __________ _________   ____  
2024-03-22T12:17:59.965725699Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T12:17:59.965728799Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T12:17:59.965731699Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T12:17:59.965734699Z         \/      \/                  \/ 
2024-03-22T12:17:59.965737799Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T12:17:59.965740499Z 
2024-03-22T12:17:59.965743399Z Documentation: http://aka.ms/webapp-linux
2024-03-22T12:17:59.965746099Z Python 3.11.7
2024-03-22T12:17:59.965748899Z Note: Any data outside '/home' is not persisted
2024-03-22T12:18:07.369198295Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T12:18:07.689948894Z App Command Line not configured, will attempt auto-detect
2024-03-22T12:18:07.759166922Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T12:18:08.273696919Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T12:18:08.292280168Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T12:18:08.311502726Z Output is compressed. Extracting it...
2024-03-22T12:18:08.311546626Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T12:18:08.341724031Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T12:18:28.803065785Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T12:18:28.823360549Z Detected an app based on Flask
2024-03-22T12:18:28.834247090Z Generating `gunicorn` command for 'app:app'
2024-03-22T12:18:28.843388409Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T12:18:29.095152984Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T12:18:29.095179784Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T12:18:30.552145233Z [2024-03-22 12:18:30 +0000] [78] [INFO] Starting gunicorn 21.2.0
2024-03-22T12:18:30.585495566Z [2024-03-22 12:18:30 +0000] [78] [INFO] Listening at: http://0.0.0.0:8000 (78)
2024-03-22T12:18:30.585533967Z [2024-03-22 12:18:30 +0000] [78] [INFO] Using worker: sync
2024-03-22T12:18:30.717696886Z [2024-03-22 12:18:30 +0000] [83] [INFO] Booting worker with pid: 83
2024-03-22T12:18:41.124235096Z [2024-03-22 12:18:41 +0000] [83] [ERROR] Exception in worker process
2024-03-22T12:18:41.124290597Z Traceback (most recent call last):
2024-03-22T12:18:41.124295397Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T12:18:41.124308197Z     worker.init_process()
2024-03-22T12:18:41.124311797Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T12:18:41.124314997Z     self.load_wsgi()
2024-03-22T12:18:41.124318097Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T12:18:41.124321197Z     self.wsgi = self.app.wsgi()
2024-03-22T12:18:41.124324097Z                 ^^^^^^^^^^^^^^^
2024-03-22T12:18:41.124333697Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T12:18:41.124337497Z     self.callable = self.load()
2024-03-22T12:18:41.124340497Z                     ^^^^^^^^^^^
2024-03-22T12:18:41.124343497Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T12:18:41.124346697Z     return self.load_wsgiapp()
2024-03-22T12:18:41.124349898Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T12:18:41.124352698Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T12:18:41.124355898Z     return util.import_app(self.app_uri)
2024-03-22T12:18:41.124358798Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:18:41.124361898Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T12:18:41.124365098Z     mod = importlib.import_module(module)
2024-03-22T12:18:41.124368198Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:18:41.124371298Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T12:18:41.124374298Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T12:18:41.124377198Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:18:41.124380198Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T12:18:41.124383698Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T12:18:41.124386898Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T12:18:41.124390098Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T12:18:41.124393298Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T12:18:41.124396398Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T12:18:41.124399498Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T12:18:41.124403698Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T12:18:41.124406598Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T12:18:41.133853323Z [2024-03-22 12:18:41 +0000] [83] [INFO] Worker exiting (pid: 83)
2024-03-22T12:18:41.946638745Z [2024-03-22 12:18:41 +0000] [78] [ERROR] Worker (pid:83) exited with code 3
2024-03-22T12:18:41.954381547Z [2024-03-22 12:18:41 +0000] [78] [ERROR] Shutting down: Master
2024-03-22T12:18:41.954668950Z [2024-03-22 12:18:41 +0000] [78] [ERROR] Reason: Worker failed to boot.

2024-03-22T12:21:19.848254207Z    _____                               
2024-03-22T12:21:19.848303708Z   /  _  \ __________ _________   ____  
2024-03-22T12:21:19.848308808Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T12:21:19.848312208Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T12:21:19.848315208Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T12:21:19.848318608Z         \/      \/                  \/ 
2024-03-22T12:21:19.848321508Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T12:21:19.848324208Z 
2024-03-22T12:21:19.848326908Z Documentation: http://aka.ms/webapp-linux
2024-03-22T12:21:19.848329708Z Python 3.11.7
2024-03-22T12:21:19.848333408Z Note: Any data outside '/home' is not persisted
2024-03-22T12:21:21.858179555Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T12:21:21.918710941Z App Command Line not configured, will attempt auto-detect
2024-03-22T12:21:21.938963303Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T12:21:22.180028732Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T12:21:22.201015905Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T12:21:22.201056405Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T12:21:22.202237020Z Output is compressed. Extracting it...
2024-03-22T12:21:22.211256337Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T12:21:49.778726484Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T12:21:49.789045721Z Detected an app based on Flask
2024-03-22T12:21:49.790181336Z Generating `gunicorn` command for 'app:app'
2024-03-22T12:21:49.799463758Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T12:21:49.979696540Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T12:21:49.980636353Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T12:21:51.407776113Z [2024-03-22 12:21:51 +0000] [78] [INFO] Starting gunicorn 21.2.0
2024-03-22T12:21:51.445954818Z [2024-03-22 12:21:51 +0000] [78] [INFO] Listening at: http://0.0.0.0:8000 (78)
2024-03-22T12:21:51.447175734Z [2024-03-22 12:21:51 +0000] [78] [INFO] Using worker: sync
2024-03-22T12:21:51.465220572Z [2024-03-22 12:21:51 +0000] [81] [INFO] Booting worker with pid: 81
2024-03-22T12:22:00.290898773Z [2024-03-22 12:22:00 +0000] [81] [ERROR] Exception in worker process
2024-03-22T12:22:00.290928774Z Traceback (most recent call last):
2024-03-22T12:22:00.290933674Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T12:22:00.290944974Z     worker.init_process()
2024-03-22T12:22:00.290948174Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T12:22:00.290951374Z     self.load_wsgi()
2024-03-22T12:22:00.290954274Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T12:22:00.290957374Z     self.wsgi = self.app.wsgi()
2024-03-22T12:22:00.290960474Z                 ^^^^^^^^^^^^^^^
2024-03-22T12:22:00.290963474Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T12:22:00.290973274Z     self.callable = self.load()
2024-03-22T12:22:00.290976274Z                     ^^^^^^^^^^^
2024-03-22T12:22:00.290979174Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T12:22:00.290982274Z     return self.load_wsgiapp()
2024-03-22T12:22:00.290985274Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T12:22:00.290988074Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T12:22:00.290991174Z     return util.import_app(self.app_uri)
2024-03-22T12:22:00.290994174Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:22:00.290997175Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T12:22:00.291000275Z     mod = importlib.import_module(module)
2024-03-22T12:22:00.291003275Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:22:00.291008875Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T12:22:00.291011875Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T12:22:00.291017875Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:22:00.291021075Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T12:22:00.291024775Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T12:22:00.291027975Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T12:22:00.291030975Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T12:22:00.291034075Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T12:22:00.291037175Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T12:22:00.291040275Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T12:22:00.291044275Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T12:22:00.291047275Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T12:22:00.298409973Z [2024-03-22 12:22:00 +0000] [81] [INFO] Worker exiting (pid: 81)
2024-03-22T12:22:00.883960235Z [2024-03-22 12:22:00 +0000] [78] [ERROR] Worker (pid:81) exited with code 3
2024-03-22T12:22:00.891798439Z [2024-03-22 12:22:00 +0000] [78] [ERROR] Shutting down: Master
2024-03-22T12:22:00.892072343Z [2024-03-22 12:22:00 +0000] [78] [ERROR] Reason: Worker failed to boot.

2024-03-22T12:32:57.874754508Z    _____                               
2024-03-22T12:32:57.874784208Z   /  _  \ __________ _________   ____  
2024-03-22T12:32:57.874787808Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T12:32:57.874790908Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T12:32:57.874793708Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T12:32:57.874803808Z         \/      \/                  \/ 
2024-03-22T12:32:57.874806908Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T12:32:57.874809608Z 
2024-03-22T12:32:57.874812208Z Documentation: http://aka.ms/webapp-linux
2024-03-22T12:32:57.874815008Z Python 3.11.7
2024-03-22T12:32:57.874817708Z Note: Any data outside '/home' is not persisted
2024-03-22T12:33:02.477953906Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T12:33:02.840223154Z App Command Line not configured, will attempt auto-detect
2024-03-22T12:33:02.848764466Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T12:33:03.340203207Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T12:33:03.362669301Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T12:33:03.420665361Z Output is compressed. Extracting it...
2024-03-22T12:33:03.420743962Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T12:33:03.420766863Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T12:33:25.734511248Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T12:33:25.753828301Z Detected an app based on Flask
2024-03-22T12:33:25.755986729Z Generating `gunicorn` command for 'app:app'
2024-03-22T12:33:25.765497654Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T12:33:25.895636558Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T12:33:25.895669859Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T12:33:27.323406059Z [2024-03-22 12:33:27 +0000] [70] [INFO] Starting gunicorn 21.2.0
2024-03-22T12:33:27.352193336Z [2024-03-22 12:33:27 +0000] [70] [INFO] Listening at: http://0.0.0.0:8000 (70)
2024-03-22T12:33:27.353610255Z [2024-03-22 12:33:27 +0000] [70] [INFO] Using worker: sync
2024-03-22T12:33:27.365925716Z [2024-03-22 12:33:27 +0000] [74] [INFO] Booting worker with pid: 74
2024-03-22T12:33:37.715320057Z [2024-03-22 12:33:37 +0000] [74] [ERROR] Exception in worker process
2024-03-22T12:33:37.715351857Z Traceback (most recent call last):
2024-03-22T12:33:37.715356957Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T12:33:37.715370058Z     worker.init_process()
2024-03-22T12:33:37.715373358Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T12:33:37.715376458Z     self.load_wsgi()
2024-03-22T12:33:37.715385258Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T12:33:37.715388558Z     self.wsgi = self.app.wsgi()
2024-03-22T12:33:37.715391458Z                 ^^^^^^^^^^^^^^^
2024-03-22T12:33:37.715394358Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T12:33:37.715397558Z     self.callable = self.load()
2024-03-22T12:33:37.715400658Z                     ^^^^^^^^^^^
2024-03-22T12:33:37.715403658Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T12:33:37.715406758Z     return self.load_wsgiapp()
2024-03-22T12:33:37.715409558Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T12:33:37.715412458Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T12:33:37.715415758Z     return util.import_app(self.app_uri)
2024-03-22T12:33:37.715418658Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:33:37.715421858Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T12:33:37.715425058Z     mod = importlib.import_module(module)
2024-03-22T12:33:37.715428058Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:33:37.715431058Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T12:33:37.715434058Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T12:33:37.715436958Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:33:37.715439959Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T12:33:37.715443359Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T12:33:37.715446659Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T12:33:37.715449759Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T12:33:37.715452959Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T12:33:37.715456159Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T12:33:37.715459159Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T12:33:37.715462959Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T12:33:37.715465959Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T12:33:37.724827581Z [2024-03-22 12:33:37 +0000] [74] [INFO] Worker exiting (pid: 74)
2024-03-22T12:33:38.365851876Z [2024-03-22 12:33:38 +0000] [70] [ERROR] Worker (pid:74) exited with code 3
2024-03-22T12:33:38.373544977Z [2024-03-22 12:33:38 +0000] [70] [ERROR] Shutting down: Master
2024-03-22T12:33:38.373915881Z [2024-03-22 12:33:38 +0000] [70] [ERROR] Reason: Worker failed to boot.

2024-03-22T12:37:27.571912017Z    _____                               
2024-03-22T12:37:27.571940118Z   /  _  \ __________ _________   ____  
2024-03-22T12:37:27.571944218Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T12:37:27.571947318Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T12:37:27.571950218Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T12:37:27.571953318Z         \/      \/                  \/ 
2024-03-22T12:37:27.571956018Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T12:37:27.571958918Z 
2024-03-22T12:37:27.571961518Z Documentation: http://aka.ms/webapp-linux
2024-03-22T12:37:27.571964218Z Python 3.11.7
2024-03-22T12:37:27.571967118Z Note: Any data outside '/home' is not persisted
2024-03-22T12:37:30.286382826Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T12:37:30.447173307Z App Command Line not configured, will attempt auto-detect
2024-03-22T12:37:30.447201008Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T12:37:30.606973173Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T12:37:30.617488636Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T12:37:30.617505636Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T12:37:30.668264319Z Output is compressed. Extracting it...
2024-03-22T12:37:30.668287920Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T12:37:52.917029012Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T12:37:52.927493043Z Detected an app based on Flask
2024-03-22T12:37:52.935277641Z Generating `gunicorn` command for 'app:app'
2024-03-22T12:37:52.942442230Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T12:37:53.055737148Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T12:37:53.055770149Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T12:37:54.133441434Z [2024-03-22 12:37:54 +0000] [70] [INFO] Starting gunicorn 21.2.0
2024-03-22T12:37:54.246375246Z [2024-03-22 12:37:54 +0000] [70] [INFO] Listening at: http://0.0.0.0:8000 (70)
2024-03-22T12:37:54.247569461Z [2024-03-22 12:37:54 +0000] [70] [INFO] Using worker: sync
2024-03-22T12:37:54.264723076Z [2024-03-22 12:37:54 +0000] [74] [INFO] Booting worker with pid: 74
2024-03-22T12:38:04.046347257Z [2024-03-22 12:38:04 +0000] [74] [ERROR] Exception in worker process
2024-03-22T12:38:04.046386258Z Traceback (most recent call last):
2024-03-22T12:38:04.046391258Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T12:38:04.046407058Z     worker.init_process()
2024-03-22T12:38:04.046417858Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T12:38:04.046421058Z     self.load_wsgi()
2024-03-22T12:38:04.046424159Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T12:38:04.046427159Z     self.wsgi = self.app.wsgi()
2024-03-22T12:38:04.046430059Z                 ^^^^^^^^^^^^^^^
2024-03-22T12:38:04.046433159Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T12:38:04.046436359Z     self.callable = self.load()
2024-03-22T12:38:04.046439359Z                     ^^^^^^^^^^^
2024-03-22T12:38:04.046442259Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T12:38:04.046445359Z     return self.load_wsgiapp()
2024-03-22T12:38:04.046448259Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T12:38:04.046451159Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T12:38:04.046454459Z     return util.import_app(self.app_uri)
2024-03-22T12:38:04.046457559Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:38:04.046460659Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T12:38:04.046463859Z     mod = importlib.import_module(module)
2024-03-22T12:38:04.046466959Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:38:04.046469859Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T12:38:04.046472859Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T12:38:04.046475859Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T12:38:04.046478759Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T12:38:04.046482459Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T12:38:04.046485559Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T12:38:04.046488759Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T12:38:04.046491860Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T12:38:04.046495060Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T12:38:04.046498160Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T12:38:04.046502660Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T12:38:04.046505660Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T12:38:04.057485515Z [2024-03-22 12:38:04 +0000] [74] [INFO] Worker exiting (pid: 74)
2024-03-22T12:38:05.367082700Z [2024-03-22 12:38:05 +0000] [70] [ERROR] Worker (pid:74) exited with code 3
2024-03-22T12:38:05.385098605Z [2024-03-22 12:38:05 +0000] [70] [ERROR] Shutting down: Master
2024-03-22T12:38:05.385345506Z [2024-03-22 12:38:05 +0000] [70] [ERROR] Reason: Worker failed to boot.

2024-03-22T13:13:29.224520491Z    _____                               
2024-03-22T13:13:29.224546991Z   /  _  \ __________ _________   ____  
2024-03-22T13:13:29.224550791Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T13:13:29.224553891Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T13:13:29.224556791Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T13:13:29.224560091Z         \/      \/                  \/ 
2024-03-22T13:13:29.224562991Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T13:13:29.224566391Z 
2024-03-22T13:13:29.224569091Z Documentation: http://aka.ms/webapp-linux
2024-03-22T13:13:29.224571791Z Python 3.11.7
2024-03-22T13:13:29.224574691Z Note: Any data outside '/home' is not persisted
2024-03-22T13:13:30.480752397Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T13:13:30.701671365Z App Command Line not configured, will attempt auto-detect
2024-03-22T13:13:30.709212953Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T13:13:30.850925100Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T13:13:30.852784722Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T13:13:30.860661213Z Output is compressed. Extracting it...
2024-03-22T13:13:30.860957517Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T13:13:30.862043229Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T13:13:52.234302029Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T13:13:52.243076147Z Detected an app based on Flask
2024-03-22T13:13:52.243089647Z Generating `gunicorn` command for 'app:app'
2024-03-22T13:13:52.252552473Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T13:13:52.367437010Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T13:13:52.367466611Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T13:13:53.469668734Z [2024-03-22 13:13:53 +0000] [71] [INFO] Starting gunicorn 21.2.0
2024-03-22T13:13:53.493374690Z [2024-03-22 13:13:53 +0000] [71] [INFO] Listening at: http://0.0.0.0:8000 (71)
2024-03-22T13:13:53.494416901Z [2024-03-22 13:13:53 +0000] [71] [INFO] Using worker: sync
2024-03-22T13:13:53.510401773Z [2024-03-22 13:13:53 +0000] [74] [INFO] Booting worker with pid: 74
2024-03-22T13:14:02.678904853Z [2024-03-22 13:14:02 +0000] [74] [ERROR] Exception in worker process
2024-03-22T13:14:02.678956254Z Traceback (most recent call last):
2024-03-22T13:14:02.678960954Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T13:14:02.678974054Z     worker.init_process()
2024-03-22T13:14:02.678977254Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T13:14:02.678980554Z     self.load_wsgi()
2024-03-22T13:14:02.678983754Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T13:14:02.678986954Z     self.wsgi = self.app.wsgi()
2024-03-22T13:14:02.678989854Z                 ^^^^^^^^^^^^^^^
2024-03-22T13:14:02.678992854Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T13:14:02.679002354Z     self.callable = self.load()
2024-03-22T13:14:02.679005554Z                     ^^^^^^^^^^^
2024-03-22T13:14:02.679008454Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T13:14:02.679011754Z     return self.load_wsgiapp()
2024-03-22T13:14:02.679014654Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T13:14:02.679017654Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T13:14:02.679020854Z     return util.import_app(self.app_uri)
2024-03-22T13:14:02.679023854Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T13:14:02.679026955Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T13:14:02.679030155Z     mod = importlib.import_module(module)
2024-03-22T13:14:02.679033155Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T13:14:02.679038755Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T13:14:02.679041855Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T13:14:02.679044855Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T13:14:02.679047755Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T13:14:02.679051255Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T13:14:02.679054455Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T13:14:02.679057555Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T13:14:02.679061055Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T13:14:02.679064355Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T13:14:02.679067355Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T13:14:02.679071555Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T13:14:02.679074455Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T13:14:02.695863169Z [2024-03-22 13:14:02 +0000] [74] [INFO] Worker exiting (pid: 74)
2024-03-22T13:14:03.431207150Z [2024-03-22 13:14:03 +0000] [71] [ERROR] Worker (pid:74) exited with code 3
2024-03-22T13:14:03.439734959Z [2024-03-22 13:14:03 +0000] [71] [ERROR] Shutting down: Master
2024-03-22T13:14:03.440018762Z [2024-03-22 13:14:03 +0000] [71] [ERROR] Reason: Worker failed to boot.

2024-03-22T14:25:00.758429708Z    _____                               
2024-03-22T14:25:00.758483309Z   /  _  \ __________ _________   ____  
2024-03-22T14:25:00.758488509Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T14:25:00.758491709Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T14:25:00.758494609Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T14:25:00.758498009Z         \/      \/                  \/ 
2024-03-22T14:25:00.758500809Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T14:25:00.758503609Z 
2024-03-22T14:25:00.758506809Z Documentation: http://aka.ms/webapp-linux
2024-03-22T14:25:00.758509609Z Python 3.11.7
2024-03-22T14:25:00.758512409Z Note: Any data outside '/home' is not persisted
2024-03-22T14:25:02.083023055Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T14:25:02.262875064Z App Command Line not configured, will attempt auto-detect
2024-03-22T14:25:02.263243669Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T14:25:02.337624765Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T14:25:02.340581105Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T14:25:02.342985437Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T14:25:02.343088038Z Output is compressed. Extracting it...
2024-03-22T14:25:02.345052365Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T14:25:24.683802505Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T14:25:24.694200241Z Detected an app based on Flask
2024-03-22T14:25:24.694224241Z Generating `gunicorn` command for 'app:app'
2024-03-22T14:25:24.704570376Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T14:25:24.826297864Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T14:25:24.826328364Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T14:25:26.213300549Z [2024-03-22 14:25:26 +0000] [70] [INFO] Starting gunicorn 21.2.0
2024-03-22T14:25:26.238633079Z [2024-03-22 14:25:26 +0000] [70] [INFO] Listening at: http://0.0.0.0:8000 (70)
2024-03-22T14:25:26.241498516Z [2024-03-22 14:25:26 +0000] [70] [INFO] Using worker: sync
2024-03-22T14:25:26.253270470Z [2024-03-22 14:25:26 +0000] [73] [INFO] Booting worker with pid: 73
2024-03-22T14:25:36.510575511Z [2024-03-22 14:25:36 +0000] [73] [ERROR] Exception in worker process
2024-03-22T14:25:36.510613911Z Traceback (most recent call last):
2024-03-22T14:25:36.510618811Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T14:25:36.510630811Z     worker.init_process()
2024-03-22T14:25:36.510634212Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T14:25:36.510637612Z     self.load_wsgi()
2024-03-22T14:25:36.510640512Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T14:25:36.510643712Z     self.wsgi = self.app.wsgi()
2024-03-22T14:25:36.510646712Z                 ^^^^^^^^^^^^^^^
2024-03-22T14:25:36.510649712Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T14:25:36.510652712Z     self.callable = self.load()
2024-03-22T14:25:36.510655912Z                     ^^^^^^^^^^^
2024-03-22T14:25:36.510659312Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T14:25:36.510662612Z     return self.load_wsgiapp()
2024-03-22T14:25:36.510665512Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T14:25:36.510668512Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T14:25:36.510671812Z     return util.import_app(self.app_uri)
2024-03-22T14:25:36.510674812Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T14:25:36.510677712Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T14:25:36.510681112Z     mod = importlib.import_module(module)
2024-03-22T14:25:36.510684212Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T14:25:36.510687312Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T14:25:36.510690612Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T14:25:36.510693612Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T14:25:36.510696612Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T14:25:36.510700512Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T14:25:36.510703712Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T14:25:36.510708513Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T14:25:36.510711713Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T14:25:36.510715013Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T14:25:36.510718113Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T14:25:36.510721913Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T14:25:36.510724913Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T14:25:36.510731813Z [2024-03-22 14:25:36 +0000] [73] [INFO] Worker exiting (pid: 73)
2024-03-22T14:25:37.364082210Z [2024-03-22 14:25:37 +0000] [70] [ERROR] Worker (pid:73) exited with code 3
2024-03-22T14:25:37.372653725Z [2024-03-22 14:25:37 +0000] [70] [ERROR] Shutting down: Master
2024-03-22T14:25:37.373146932Z [2024-03-22 14:25:37 +0000] [70] [ERROR] Reason: Worker failed to boot.

2024-03-22T15:17:18.619296677Z    _____                               
2024-03-22T15:17:18.619329578Z   /  _  \ __________ _________   ____  
2024-03-22T15:17:18.619333578Z  /  /_\  \\___   /  |  \_  __ \_/ __ \ 
2024-03-22T15:17:18.619336778Z /    |    \/    /|  |  /|  | \/\  ___/ 
2024-03-22T15:17:18.619339878Z \____|__  /_____ \____/ |__|    \___  >
2024-03-22T15:17:18.619343078Z         \/      \/                  \/ 
2024-03-22T15:17:18.619345878Z A P P   S E R V I C E   O N   L I N U X
2024-03-22T15:17:18.619348678Z 
2024-03-22T15:17:18.619351278Z Documentation: http://aka.ms/webapp-linux
2024-03-22T15:17:18.619354078Z Python 3.11.7
2024-03-22T15:17:18.619356678Z Note: Any data outside '/home' is not persisted
2024-03-22T15:17:20.015947054Z Starting OpenBSD Secure Shell server: sshd.
2024-03-22T15:17:20.226910930Z App Command Line not configured, will attempt auto-detect
2024-03-22T15:17:20.228552652Z Launching oryx with: create-script -appPath /home/site/wwwroot -output /opt/startup/startup.sh -virtualEnvName antenv -defaultApp /opt/defaultsite
2024-03-22T15:17:20.437755604Z Found build manifest file at '/home/site/wwwroot/oryx-manifest.toml'. Deserializing it...
2024-03-22T15:17:20.439963933Z Build Operation ID: 75ef9913c02ffce1
2024-03-22T15:17:20.478931046Z Oryx Version: 0.2.20240127.1, Commit: 4b7f2dffcc69c214f9806d67a85ec8926e2393e1, ReleaseTagName: 20240127.1
2024-03-22T15:17:20.478963346Z Output is compressed. Extracting it...
2024-03-22T15:17:20.478968346Z Extracting '/home/site/wwwroot/output.tar.gz' to directory '/tmp/8dc4a5f33ada8c2'...
2024-03-22T15:17:43.065850400Z App path is set to '/tmp/8dc4a5f33ada8c2'
2024-03-22T15:17:43.089801817Z Detected an app based on Flask
2024-03-22T15:17:43.089816317Z Generating `gunicorn` command for 'app:app'
2024-03-22T15:17:43.083754937Z Writing output script to '/opt/startup/startup.sh'
2024-03-22T15:17:43.194146497Z Using packages from virtual environment antenv located at /tmp/8dc4a5f33ada8c2/antenv.
2024-03-22T15:17:43.194175597Z Updated PYTHONPATH to '/opt/startup/app_logs:/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages'
2024-03-22T15:17:44.469749469Z [2024-03-22 15:17:44 +0000] [72] [INFO] Starting gunicorn 21.2.0
2024-03-22T15:17:44.532951205Z [2024-03-22 15:17:44 +0000] [72] [INFO] Listening at: http://0.0.0.0:8000 (72)
2024-03-22T15:17:44.533574913Z [2024-03-22 15:17:44 +0000] [72] [INFO] Using worker: sync
2024-03-22T15:17:44.541343016Z [2024-03-22 15:17:44 +0000] [75] [INFO] Booting worker with pid: 75
2024-03-22T15:17:53.256897264Z [2024-03-22 15:17:53 +0000] [75] [ERROR] Exception in worker process
2024-03-22T15:17:53.256948065Z Traceback (most recent call last):
2024-03-22T15:17:53.256952765Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
2024-03-22T15:17:53.256965065Z     worker.init_process()
2024-03-22T15:17:53.256968365Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
2024-03-22T15:17:53.256971865Z     self.load_wsgi()
2024-03-22T15:17:53.256974665Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2024-03-22T15:17:53.256977865Z     self.wsgi = self.app.wsgi()
2024-03-22T15:17:53.256980665Z                 ^^^^^^^^^^^^^^^
2024-03-22T15:17:53.256983665Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
2024-03-22T15:17:53.256986765Z     self.callable = self.load()
2024-03-22T15:17:53.256995165Z                     ^^^^^^^^^^^
2024-03-22T15:17:53.256998565Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2024-03-22T15:17:53.257001665Z     return self.load_wsgiapp()
2024-03-22T15:17:53.257004566Z            ^^^^^^^^^^^^^^^^^^^
2024-03-22T15:17:53.257007366Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2024-03-22T15:17:53.257010466Z     return util.import_app(self.app_uri)
2024-03-22T15:17:53.257013466Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T15:17:53.257016466Z   File "/tmp/8dc4a5f33ada8c2/antenv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
2024-03-22T15:17:53.257019766Z     mod = importlib.import_module(module)
2024-03-22T15:17:53.257022766Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T15:17:53.257025766Z   File "/opt/python/3.11.7/lib/python3.11/importlib/__init__.py", line 126, in import_module
2024-03-22T15:17:53.257028766Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-03-22T15:17:53.257031666Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-03-22T15:17:53.257034666Z   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
2024-03-22T15:17:53.257038066Z   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
2024-03-22T15:17:53.257041266Z   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
2024-03-22T15:17:53.257044366Z   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
2024-03-22T15:17:53.257047466Z   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
2024-03-22T15:17:53.257050566Z   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2024-03-22T15:17:53.257053566Z   File "/tmp/8dc4a5f33ada8c2/app.py", line 10, in <module>
2024-03-22T15:17:53.257058066Z     from applicationinsights.flask.ext import AppInsights
2024-03-22T15:17:53.257061066Z ModuleNotFoundError: No module named 'applicationinsights'
2024-03-22T15:17:53.264629867Z [2024-03-22 15:17:53 +0000] [75] [INFO] Worker exiting (pid: 75)
2024-03-22T15:17:53.778236320Z [2024-03-22 15:17:53 +0000] [72] [ERROR] Worker (pid:75) exited with code 3
2024-03-22T15:17:53.779370035Z [2024-03-22 15:17:53 +0000] [72] [ERROR] Shutting down: Master
2024-03-22T15:17:53.797957983Z [2024-03-22 15:17:53 +0000] [72] [ERROR] Reason: Worker failed to boot.

